compile/compile.go

```diff
@@ -483,8 +486,28 @@ func (c *Compiler) compilePlan(ctx context.Context) error {
 
 	// Prepare modules and builtins for the planner.
 	modules := []*ast.Module{}
+	allPaths := ast.NewSet()
 	for _, module := range c.compiler.Modules {
 		modules = append(modules, module)
+
+		modulePath, err := module.Package.Path.Ptr()
+		if err != nil {
+			return err
+		}
+		allPaths.Add(ast.StringTerm(modulePath))
+
+		for _, rule := range module.Rules {
+			rulePath, err := rule.Path().Ptr()
+			if err != nil {
+				return err
+			}
+			allPaths.Add(ast.StringTerm(rulePath))
+		}
+	}
+
+	unmatchedQueryPaths := queryPaths.Diff(allPaths)
+	if unmatchedQueryPaths.Len() > 0 {
+		return fmt.Errorf("entrypoint(s) %v unmatched by any existing path in modules", unmatchedQueryPaths.String())
```

Comment:
nit: `entrypoint %v does not refer to a rule|decision|policy decision`

Not sure what term we should use but I think we should avoid saying "any existing path in modules". "rule" is probably the most understood.

Reply:
Updated to something similar. It'll be singular now, and will just report on the first failure... but OTOH, chances should be pretty slim that someone provides a bunch of unmapped entrypoints.. and if they do, they would likely learn from the first error reported.. or what do you think?